docs/as-a-data-transfer-object/collections.md

```diff
@@ -232,9 +235,9 @@ SongData::collect(Song::all(), DataCollection::class)->first(); // SongData obje
 In previous versions of the package it was possible to use the `collection` method to create a collection of data objects:
 
 ```php
-SongData::collect(Song::all()); // returns a DataCollection of SongData objects
-SongData::collect(Song::paginate()); // returns a PaginatedDataCollection of SongData objects
-SongData::collect(Song::cursorPaginate()); // returns a CursorPaginatedCollection of SongData objects
+SongData::collection(Song::all()); // returns a DataCollection of SongData objects
+SongData::collection(Song::paginate()); // returns a PaginatedDataCollection of SongData objects
+SongData::collection(Song::cursorPaginate()); // returns a CursorPaginatedCollection of SongData objects
```

Comment:
@rubenvanassche or anyone who will be reviewing this.
Just to point your attention here.
I haven't used the previous versions of this pacakge, so I was not sure if this change was correct or not.

Reply:
Totally correct!